test/libsolidity/analysis/FunctionCallGraph.cpp

```diff
@@ -1693,8 +1693,8 @@ BOOST_AUTO_TEST_CASE(using_for)
 			{"Entry", "function L.ext(struct S)"},
 		}},
 		{"C", {
-			{"Entry", "function C.test()"},
-			{"function C.test()", "function L.inr(struct S)"},
+			{"Entry", "function C.pub()"},
+			{"function C.pub()", "function L.inr(struct S)"},
```

Comment:
My usual convention in these tests was to make the visibility obvious in the name. For some reason I did not do it in this particular test.